backends/edk2-compat/edk2-svc-generate.c

```diff
@@ -448,9 +384,9 @@ static int getOutputData(const unsigned char *buff, size_t size, struct Argument
 			rc = generateHash(buff, size, args, hashFunction, outBuff, outBuffSize);
 			break;
         case 'x':
-            //intentional flow
+            // intentional flow
```

Comment:
not a high priority, but if you use something like `/* fall through */`, gcc will recognise it and not fire a warn when compiled with `-Wimplicit-fallthrough=N`: see https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html and https://developers.redhat.com/blog/2017/03/10/wimplicit-fallthrough-in-gcc-7/

You don't need to change this now: any comment is better than no comment. A future patch can make sure secvarctl compiles with -Wimplicit-fallthrough.